src/requirements/data/majors/info.ts

```diff
@@ -58,8 +58,208 @@ const infoRequirements: readonly CollegeOrMajorRequirement[] = [
+        checker: includesWithSubRequirements(
+          ['INFO 3300', 'INFO 3950', 'CS 4780', 'CS 4786', 'ORIE 3120', 'ORIE 4740', 'STSCI 4740'],
+          ['INFO 2770', 'INFO 3350', 'INFO 4100', 'INFO 4120', 'INFO 4130', 'INFO 4300', 'CS 4740'],
+          [
+            'INFO 3200',
+            'INFO 3561',
+            'INFO 4200',
+            'INFO 4240',
+            'INFO 4250',
+            'INFO 4270',
+            'INFO 4561',
+            '​INFO 4940',
+            'COMM 4242',
+            'ENGL 3778',
+            'STS 3440',
+          ],
+          ['INFO 4310', 'COMM 3150', 'COMM 3189', 'COMM 4200', 'COMM 4860', 'GOVT 2169', 'SOC 3580']
+        ),
+        perSlotMinCount: [1, 1, 1, 1],
+      },
+      'Digital Culture and Production': {
+        description:
+          'A. Digital Culture and History, B. Digital Production, C. Media, Art, Design. ' +
+          'One course each from A and B; 2 additional courses from A or C.',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['INFO 2921', 'INFO 3200', 'INFO 3561', 'STS 3440', 'STS 4040'],
+          ['​INFO 2300', 'INFO 3152', 'INFO 3300', 'INFO 4320', 'CS 3758', 'CS 4620'],
+          [
+            // A
+            'INFO 2921',
+            'INFO 3200',
+            'INFO 3561',
+            'STS 3440',
+            'STS 4040',
+            // C
+            'INFO 2750',
+            'INFO 3450',
+            'INFO 3660',
+            'INFO 4152',
+            'INFO 4240',
+            'INFO 4400',
+            'INFO 4420',
+            '​INFO 4940',
+            '​INFO 4940',
+            'ART 3705',
+            'ARTH 4151',
+            'ARTH 4154',
+            'COML 3115',
+            '​ENGL 3741',
+            'HIST 2293',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 2],
+      },
+      'Information Ethics, Law, and Policy': {
+        description:
+          'A. Frameworks and Institutions (choose one), ' +
+          'B. Methods and Analysis (choose one), ' +
+          'C. Cases/Topics (choose one), ' +
+          'D. Tools and Technical Domains (choose one)',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['INFO 4113', 'INFO 4200', 'INFO 4250', 'INFO 4301', 'HADM 4890', 'STS 2761'],
+          ['INFO 4240', 'INFO 4800', '​INFO 4940', 'COMM 4242', 'CRP 3210', 'PAM 2300'],
+          [
+            'INFO 3200',
+            'INFO 3460',
+            'INFO 3561',
+            'INFO 4270',
+            'INFO 4561',
+            'INFO 4940',
+            'STS 3440',
+            'STS 4040',
+          ],
+          ['INFO 3300', 'INFO 3350', 'INFO 4120', 'INFO 4130', 'INFO 4300']
+        ),
+        perSlotMinCount: [1, 1, 1, 1],
+      },
+      'Interactive Technologies': {
+        description:
+          'A. Building with Hardware (choose one), ' +
+          'B. Working with Data/Software (choose one), ' +
+          'C. Context/Application Domains (choose one)',
```

Comment:
Added a comment on this. It turns out it's not easy, since adding CS 2110 here might introduce double counting warning? I added a TODO here.

Reply:
Thanks! Is this something that can be done before launch or would this be more disruptive since we would be changing the format of the toggleable reqs?

Reply:
@tcho6319 It should be pretty straightforward. It requires a slight tweak in requirement algorithm. I might do this live for devs who wants to learn more about requirement algorithm.